lib/src/image_dart_compiling/transformer.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:barback/barback.dart';
+import 'package:path/path.dart' show url;
+import 'dart:async';
+import 'package:scissors/src/utils/io_utils.dart' show readAll;
+import 'package:scissors/src/image_dart_compiling/image_dart_compiling.dart';
+import '../image_inlining/image_inliner.dart';
+
+const outputFileName = 'images.dart';
+
+/// Transformer that compiles several image files to a Dart source file with
```

Comment:
Nit: could you explain somewhere (here, in readme.md or both) that it groups images by directory, which formats it supports, etc.